src/system-report/system-report.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, Get, Param, Post, UseGuards } from '@nestjs/common';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { SystemReportService } from './system-report.service';
+import { SystemReport } from '../model/system-report.model';
+import { AuthGuard } from '../guards/auth.guard';
+
+@ApiBearerAuth()
+@ApiTags('Report')
+@Controller('report/system')
+export class SystemReportController {
+    constructor(private readonly service: SystemReportService) {}
+    @UseGuards(AuthGuard)
```

Comment:
You can use `@UseGuards(AuthGuard)` before controller if expect to use it for every endpoints.